src/index.ts

```diff
@@ -1,12 +1,28 @@
-import express, { Request, Response } from 'express';
+import express from 'express';
+
+import { generateCronExpression, generateReadableExpression } from './cronGenerator';
 
 const app = express();
 const port = 3000;
 
-app.get('/', (req: Request, res: Response) => {
-  res.send('Hello World!');
+app.get('/v1/generate', (req, res) => {
```

Comment:
Define a `controller/routes.ts` file.
Define your controller function `(req, res) => {...}` in a separate file `controllers/cron.ts`.

`app.js` should only contain logic relevant to the express server and a reference to the router, nothing else.

Reply:
Or better yet, as per the jest docs say. Separating the concerns for the server, app, router, controllers.